api/src/main/java/io/strimzi/api/kafka/model/KafkaUserQuotas.java

```diff
@@ -34,6 +34,17 @@
     private Integer producerByteRate;
     private Integer consumerByteRate;
     private Integer requestPercentage;
+    private Double controllerMutationRate;
+
+    @Description(" The quota on rate at which mutations are accepted for the create topics request, the create partitions request and the delete topics request. The rate is accumulated by the number of partitions created or deleted.")
```

Comment:
I would just leave the description as stated in KIP-599: 
```
The rate at which mutations are accepted for the create topics request, the create partitions request and the delete topics request. The rate is accumulated by the number of partitions created or delet
```

Reply:
I would change this in all the descriptions and docs below. Unless "the quota on rate" means something specific that I am missing?

Reply:
Actually I added it for consistency since every other quota was using ''A quota on" in description

Reply:
Ok, then I would change it to `A quota on the rate.....` (also get rid of the leading whitespace)